lms/services/digest.py

```diff
@@ -309,11 +309,7 @@ def _is_instructor(
                         (course.id for course in courses)
                     )
                 )
-                .filter(
-                    AssignmentMembership.user_id.in_(
-                        user.id for user in unified_user.users
-                    )
-                )
```

Comment:
This was the only place where `UnifiedUser.users` was actually used and it was to get the `id`'s only